example/app/example-app/run.go

```diff
@@ -78,7 +78,7 @@ func run(args []string) error {
 
 			&cli.StringFlag{
 				Name:    "tendermintURL",
-				Usage:   "Tendermint HTTP RPC URL",
+				Usage:   "TendermintApp HTTP RPC URL",
```

Comment:
Is this change on purpose?

Reply:
Should be the IDE refactor effect. Changed back.